src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -988,6 +988,16 @@ const TorrentsTable = new Class({
         };
 
         // name, category
+        this.columns['name'].compareRows = function(row1, row2) {
```

Comment:
perhaps you can use `String.prototype.localeCompare()`:
```js
this.columns['name'].compareRows = function(row1, row2) {
    const row1Val = this.getRowValue(row1);  // use camel case 
    const row2Val = this.getRowValue(row2);  // use const
    return row1Val.localeCompare(row2Val, undefined, {numeric: true, sensitivity: 'base'});
};
```

Also it should be a good idea to separate out the comparison function as it might be useful to other fields.

Reply:
The reason why I still used `toLocaleLowerCase()` is that `localeCompare()` has slightly lower compatibility: [some browsers don't support its parameters](https://caniuse.com/#feat=localecompare). However, if you insist, I can of course do the change. It's fine to me.

As for separating, I consider no other fields need case-insensitivity and locale, as their strings are generated by qbittorrent, or should be case-sensitive (e.g. path, category, tags), not like arbitrary torrent names. So, it may be not necessary to separate.

Reply:
>The reason why I still used toLocaleLowerCase() is that localeCompare() has slightly lower compatibility: some browsers don't support its parameters

We don't have those browsers on our support list, so it is fine to ignore them.

>As for separating, I consider no other fields need case-insensitivity and locale, as their strings are generated by qbittorrent, or should be case-sensitive (e.g. path, category, tags), not like arbitrary torrent names. So, it may be not necessary to separate.

Most of the time WebUI follows GUI design and "tag", "category" fields are using case-insensitive comparison in GUI.